lib/services/auth.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:brew_crew/user_id_model/user.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AuthService {
+  final FirebaseAuth auth = FirebaseAuth.instance;
+
+  User userData;
```

Comment:
jest jakaś dziwna zależność międy AuthService a DatabaseService co do tego usera. zdecyduj się kto nim zarządza i przenieś całą logikę do jednego serwisu. (DatabaseService wygląda na lepszy wybór)

Reply:
to jak jest teraz gadałem z Jakobem i wydawaje sie być w porządku, serwis auth odpowiada za autoryzacje, databse za operacje na danych